packages/app/src/elements/import-dialog.ts

```diff
@@ -40,14 +97,126 @@ export class ImportDialog extends Dialog<File, void> {
                 ></pl-select>
 
                 <div class="small padded" ?hidden=${this._formatSelect && this._formatSelect.value !== imp.CSV.value}>
-                    ${$l(
-                        "IMPORTANT: Before importing, please make sure that your CSV data " +
-                            "is structured according to {0}'s specific requirements!",
-                        process.env.PL_APP_NAME!
-                    )}
-                    <a href="#" @click=${this._downloadCSVSampleFile}> ${$l("Download Sample File")} </a>
+                    ${$l("Choose the correct column names and types for each column below. If you are having trouble,")}
+                    <a href="#" @click=${this._downloadCSVSampleFile}> ${$l("Download the Sample File")} </a>.
                 </div>
 
+                <pl-scroller
+                    class="stretch"
+                    ?hidden=${this._formatSelect && this._formatSelect.value !== imp.CSV.value}
+                >
+                    <ul class="vertical spacing layout">
+                        <div class="padded">
+                            <label>
+                                <input type="checkbox" id="csvHasDataOnFirstRowCheckbox" />
+                                ${$l("Check if the CSV file has data on the first row (instead of the column names)")}
+                            </label>
```

Comment:
Please use `<pl-toggle-button></pl-toggle-button>` for this to make it more consistent with the rest of the UI. Also, for the label I'd prefer "First row contains field names" which should be checked by default.